FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Extensions/MetricsLoggerExtensions.cs

```diff
@@ -32,7 +32,8 @@ public static void LogTotalErrorsMetrics(this IMetricsLogger metricsLogger, Exce
                     ProcessJobExecutionError(metricsLogger, ex, message);
                     break;
                 default:
-                    break;
+                    throw new ConfigurationErrorException(
```

Comment:
Can we add some logs for metrics? like what has been logged and what kinds of error has been thrown. Which can help us diagnose the errors.

Reply:
We have logged error information outside every time call logErrorMetrics API.  Do you mean add logs for other metrics? (e.g. log successful resource count or latency)

Reply:
And we also log detailed metrics information in MetricsLogger.

Reply:
I mean whether we have a good way to verify the metric information we sent since the metrics logger is expected to be implemented by the outside runner (external metrics provider).  Then we can compare the result if the metrics looks odd (might be error from the external metricslogger implemented by errors).
Add log every time when call logErrorMetrics api is good. But seems there are many cases that metrics has not been logged like in FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/HealthCheckBackgroundService.cs

Reply:
Thanks for the suggestions, added logs every time call log metrics.